README.md

```diff
@@ -4,14 +4,62 @@ Library for managing [WebMock][] fixtures.
 
 [WebMock]: https://github.com/bblimke/webmock
 
-## Installation
+[Documentation](http://www.rubydoc.info/github/underdogio/webmock-fixtures).
 
+## Installation
 `gem install webmock-fixtures`
 
 or with Bundler
 
 `gem 'webmock-fixtures`
 
+## Getting Started
+WebMock fixtures provides a way to pre-define all of your fixtures ahead of time and start them easily during tests.
+
+```ruby
+require "webmock/fixtures"
+
+# Register a fixture as `:get_example`
+WebMock::Fixtures::Manager.register_fixture(:get_example, :get, %r{www.example.org}, :body => "Hello World")
+
+RSpec.describe MyApp do
```

Comment:
In most scenarios, we should be writing "Getting Started" examples such that they can be run. Instead of using something like `sinatra`. How about we make this about a generic HTTP request?

Here's what we have in `httpretty-fixtures`:

https://github.com/underdogio/httpretty-fixtures/blob/2.1.0/docs/getting_started.py